app/root.tsx

```diff
@@ -253,10 +253,10 @@ export function Layout({ children }: { children?: ReactNode }) {
         <DevBanner />
         <Header />
         <NewsBanner>
-          New! October 18 GCN Classic Outage and Schema v4.2.0. See{' '}
```

Comment:
The cheery exclamation "New!" is inappropriate here.

The noun phrase "Legacy Circular Email Retirement" is structurally ambiguous.